File: abc/abc179/b.py
# https://atcoder.jp/contests/abc179/submissions/37705104

n = int(input())
ans = False
cnt = 0

for _ in range(n):
  a, b = map(int, input().split())
  if a == b:
    cnt += 1
  else:
    cnt = 0

  if cnt >= 3:
    ans = True

print("Yes" if ans else "No")

File: abc/abc179/b.cpp
// https://atcoder.jp/contests/abc179/submissions/27202365

#include <bits/stdc++.h>
using namespace std;
int main() {
  int d, e, n, count = 0; cin >> n;
  string ans = "No";
  while(cin >> d >> e) {
    if(d == e) count++;
    else count = 0;
    if(count == 3) ans = "Yes";
  }
  cout << ans;
}
